CONTRIBUTING.md

```diff
@@ -51,7 +51,9 @@ more details.
 ### Run Steps
 
 > Note: requires [Node.js](https://nodejs.org/en/download/) v14
-> and [npm](https://www.npmjs.com/package/npm) v6.14
+> and [npm](https://www.npmjs.com/package/npm) v6.14. The quickest way to set these up is to install
```

Comment:
Maybe there's a slightly different way of wording this that doesn't imply that nvm is the quickest and/or best way of managing node versions while still pointing out that it may be convenient for contributors?

I don't use it myself but I think this small change to the docs a helpful addition to facilitate a more efficient first experience with the project :)

Reply:
That's a good point. How about something like "an easy way to set these up"? That doesn't give a total value judgment, but it still provides a recommendation for people getting up to speed.

Reply:
Sounds good to me.

Reply:
Pushed the edit!